arkouda/io.py

```diff
@@ -1565,5 +1611,7 @@ def read(
             strict_types=strictTypes,
             allow_errors=allow_errors,
         )
+    elif ftype.lower() == "csv":
+        return read_csv(filenames, datasets=datasets, column_delim=",", allow_errors=allow_errors)
```

Comment:
this should be `column_delim=column_delim` right?

Reply:
Yes. Initially, I was going to have these use the default and not add the parameter to the functions, but changed my mind and forgot to update here.